_inc/lib/jetpack-wpes-query-builder/jetpack-wpes-query-builder.php

```diff
@@ -338,4 +338,33 @@ public function build_aggregation() {
 		return $aggregations;
 	}
 
+	/**
+	 * Assemble the 'suggest' portion of an ES query, using a phrase suggester. Note that this is experimental and
+	 * only works with English tokens.
+	 *
+	 * @param string $query The query text being searched.
+	 *
+	 * @return array An experimental suggest query.
+	 */
+	public function build_experimental_suggest( $query ) {
+		return array(
+			'text' => $query,
+			'experimental-phrase-suggester' => array(
+				'phrase' => array(
+					'field'                      => 'mlt_content.default',
```

Comment:
This gets kinda complicated, but you should be able to use logic similar to https://github.com/Automattic/jetpack/blob/master/modules/search/class.jetpack-search.php#L825 to select the language specific field (assuming that makes sense). Definitely also need to handle when a VIP index is specified in which case the field is "mlt_content" rather than "mlt_content.default".